### ISSUE TITLE
Windows-specific process cleanup issues

### ISSUE THREAD

**Issue by alexandrul** [WPO-Foundation/wptagent#74]
On my server wptagent tries to log off the current user :smile:

The corresponding log line is:

    12:58:50.805 - Closing Window: Start (Button) - 52x38 : 1

Is it possible to add a config option to just list the processes and skip their termination?
Also, would it be ok to whitelist the window title?

**Comment by pmeenan** [comment]
Sorry about that.  I put in a quick hack to not close "Button" class windows.  I'll rewrite the logic to enumerate windows that are children of the desktop which will hopefully avoid any of the taskbar windows.  What version of Windows were you running on (I've been testing on 10).

**Comment by pmeenan** [comment]
Turns out it wasn't quite that easy because they are still children of the desktop.  I added logic to get the exe running the process for the Windows and added a whitelist for exe's to prevent closing explorer.exe (also added a title whitelist to prevent closing start which should help as a safety).

The log message also now spits out the exe name of the process so if we get more problematic ones we can add them to the appropriate whitelist.
